src/main/java/core/basesyntax/strategy/handler/IncrementHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+
+public class IncrementHandler implements OperationHandler {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line before the class declaration.